README-dev.md

```diff
@@ -33,6 +33,12 @@ Let's say the published package version number 0.0.15, verify that it is install
 /tmp/tmp1 $ npm install @celo/cli@0.0.20
 ```
 
+Add a tag with the git commit of the published version
+
+```
+$ npm dist-tag add <package-name>@<version> [<tag>]
```

Comment:
since this is happening in a PR, and the commits get squashed after that.. probably won't serve the purpose you want

Reply:
The latest commit on a branch corresponds to the latest PR that was merged in- for example, I tagged contactkit@0.1.2 with the commit hash `3c1fe1b`, which corresponds to master on 9/29 (see here https://github.com/celo-org/celo-monorepo/commits/master) 

When we publish to npm I would expect we use a branch with changes that have already been merged in (rather than an open PR), so we can just use the latest commit hash.